src/components/OrderSummary/TotalPrice.js

```diff
@@ -2,22 +2,19 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
-import { fontSmall } from '../../styles/designTokens';
-import PriceBadge from './PriceBadge';
-import PriceRow from './PriceRow';
+import { fontSmall, fontWeightBold } from '../../styles/designTokens';
+import Row from './TextRow';
 
 const PriceSummary = styled.span`
   font-size: ${fontSmall};
-  font-weight: bold;
+  font-weight: ${fontWeightBold};
 `;
 
 function TotalPrice({ price }) {
   return (
-    <PriceRow>
-      <PriceSummary>
-        TOTAL<PriceBadge>${price}</PriceBadge>
-      </PriceSummary>
-    </PriceRow>
+    <Row>
+      <PriceSummary>TOTAL ${price}</PriceSummary>
```

Comment:
Did you resign to right align the `Total` $ price?

Reply:
Will update styles for it next PR.